netbox/templates/base/layout.html

```diff
@@ -51,8 +51,20 @@ <h1 class="navbar-brand pb-0">
         {# Navigation menu #}
         <div class="collapse navbar-collapse" id="sidebar-menu">
           {% nav %}
-        </div>
 
+          {# Release info #}
+          <div class="text-muted text-center fs-5 my-3">
+            {{ settings.RELEASE.name }}
+            {% if not settings.RELEASE.features.commercial %}
+              <div>
+                {% trans "Get" %}
```

Comment:
very minor nit: I'd either put a colon after Get or else include it in the link as it's not obvious they are links and I first hovered over Get.  Otherwise everything is good.